src/Bridge/RamseyUuid/Identifier/Normalizer/UuidNormalizer.php

```diff
@@ -28,20 +28,22 @@ final class UuidNormalizer implements DenormalizerInterface
 {
     /**
      * {@inheritdoc}
+     *
+     * @throws NotNormalizableValueException Occurs when the identifier format is invalid
      */
     public function denormalize($data, $class, $format = null, array $context = [])
     {
         try {
             return Uuid::fromString($data);
         } catch (InvalidUuidStringException $e) {
-            throw new InvalidIdentifierException($e->getMessage(), $e->getCode(), $e);
+            throw new NotNormalizableValueException('Not found, because of an invalid identifier format', 404, $e);
         }
     }
 
     /**
      * {@inheritdoc}
      */
-    public function supportsDenormalization($data, $type, $format = null)
+    public function supportsDenormalization($data, $type, $format = null): bool
```

Comment:
you can't change the return type here it's not in the interface

Reply:
I'll remove it with my next commit. Thank you! 😃

Reply:
It's safe to add the return type here. :smile: